share/openPMD/cmake/FindADIOS.cmake

```diff
@@ -107,7 +107,9 @@ if(ADIOS_FIND_COMPONENTS)
         elseif(comp STREQUAL "sequential")
             set(OPTLIST "${OPTLIST}s")
         else()
-            message("ADIOS component ${COMP} is not supported. Please use fortran, readonly, or sequential")
+            if(NOT ADIOS_FIND_QUIETLY)
+                message("ADIOS component ${COMP} is not supported. Please use fortran, readonly, or sequential")
```

Comment:
This is potentially a case where we could signal ```FATAL_ERROR```.
The user is requesting a component we will never be able to provide.

Reply:
Oh, quite the contrary! We are inside a `Find*.cmake` script. The worst thing it is allowed to do is *not find* what is requested.

Fatal errors are controlled by setting `REQUIRED` in `find_package` or by handling the `Package_FOUND` variable manually.

So we are all set :-)